internal/aws/k8scommon/k8sclient/clientset.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This might not be acceptable to this repository.

Reply:
Thanks! I will change the license and improve the test coverage. Do you have any concern about the size of the PR? Should I split it into smaller ones?